core/src/test/java/org/infinispan/scattered/ScatteredSyncFuncTest.java

```diff
@@ -188,4 +191,19 @@ protected void assertLocalValue(int node, MagicKey key, String expectedValue) {
       assertNotNull(ice);
       assertEquals(expectedValue, ice.getValue());
    }
+
+   @Override
+   public void testMergeFromNonOwner() {
+      // TODO : Add support for ST in functional commands : https://issues.jboss.org/browse/ISPN-8078
```

Comment:
You keep using "ST" for scattered cache - that's a bit confusing, ST usually denotes state transfer